README.md

```diff
@@ -110,37 +110,40 @@ Otherwise :violin: kindly ask someone from the dev team to help troubleshoot.
+|-------------------------------------------------|----------------------------------------| -----------
+| DEFAULT_WEBSITE_URL                             | "http://localhost:8080"                | Service URL
+| DEFAULT_LEGACY_CACHE_API_URL                    | "http://localhost:29100"               | Cache API Service URL
+| DEFAULT_SLACK_WARNING_CHANNEL                   | "slack-client-test"                    | Slack channel
+| DEFAULT_SLACK_ALARM_CHANNEL                     | "slack-client-test"                    | Slack alarm channel
+| DEFAULT_SLACK_USERNAME                          | "Zebedee"                              | Slack user
+| DEFAULT_PUBLIC_WEBSITE_URL                      | "http://localhost:8080"                | Service public URL
+| DEFAULT_FLORENCE_URL                            | "http://localhost:8081"                | Florence URL
+| DEFAULT_BRIAN_URL                               | "http://localhost:8083"                | Brian URL
+| DEFAULT_TRAIN_URL                               | "http://localhost:8084"                | Train URL
+| DEFAULT_DYLAN_URL                               | "http://localhost:8085"                | Dylan URL
+| CONTENT_DIRECTORY                               | "content"                              | Content directory
+| KAFKA_SEC_PROTO                                 | _unset_                                | if set to "TLS", kafka connections will use TLS
+| KAFKA_SEC_CLIENT_CERT                           | _unset_                                | PEM for the client certificate [1]
+| KAFKA_SEC_CLIENT_KEY                            | _unset_                                | PEM for the client key [1]
+| MATHJAX_SERVICE_URL                             | "http://localhost:8888"                | Mathjax service URL
+| DATASET_API_URL                                 | "http://localhost:22000"               | Dataset API URL
+| IMAGE_API_URL                                   | "http://localhost:24700"               | Image API URL
+| ENABLE_KAFKA                                    | false                                  | Feature flag to send kafka messages when a collection is published
+| ENABLE_LEGACY_CACHE_API                         | false                                  | Feature flag to enable Cache API
+| KAFKA_ADDR                                      | "localhost:9092"                       | Comma seperated list of kafka brokers
+| KAFKA_CONTENT_UPDATED_TOPIC                     | content-updated                        | Kafka topic to send content updated event to
+| DATASET_API_AUTH_TOKEN                          | "FD0108EA-825D-411C-9B1D-41EF7727F465" | Dataset API authentication token
+| SERVICE_AUTH_TOKEN                              | "15C0E4EE-777F-4C61-8CDB-2898CEB34657" | Service API authentication token
+| LEGACY_CACHE_API_SERVICE_AUTH_TOKEN             | "...."                                 | Legacy Cache API authentication token
```

Comment:
That's not the default value that's being set in the config.

Reply:
Not sure this is actually right to put a token here. I think we should have the production token here as everything in this branch should be set to work in production. I suggest this has to be updated by ONS at some point later when its going in production? Ideally we read the token from the environment and don't put a token here as tokens should not be stored in github or any version control.

Reply:
You cannot put a production token here, because that would be a vulnerability. I assume that the other tokens listed here are, like the Dataset API or the Service one, are development only tokens. I agree with you in that we shouldn't even be storing the tokens anywhere in the repo, but you have already committed your token anyway, it's in `Configuration.java`, so if it's OK to set it there, it's OK to set it here.

Reply:
the token I committed is my local development token. So this is not a production token.

Reply:
Cool, but you initially said "I think we should have the production token here", hence my comment.